### DISCUSSION TITLE
Commenting a blade view in Laravel app doesn't work

### DISCUSSION THREAD

**Discussion by glxghr** [laravel/framework#49822] [Q&A]
### Laravel Version

10.18.0

### PHP Version

8.0.2

### Description

I have blade navbar view that I tried to comment by selecting all of it but it didn't work. You have to erase all of it.

### Steps To Reproduce

Simply try commenting a blade view template having some nav, div, etc. html tags with a script tag at the end.

**Comment by rojtjo** [comment]
How are you commenting the template?

It's working fine for me when I wrap all content in `{{--` and `--}}` tags.
